src/controllers/publish.ts

```diff
@@ -570,106 +570,83 @@ export const downloadDataset = async (req: Request, res: Response, next: NextFun
   }
 };
 
-export const redirectToTasklist = (req: Request, res: Response) => {
-  res.redirect(req.buildUrl(`/publish/${req.params.datasetId}/tasklist`, req.language));
+export const redirectToOverview = (req: Request, res: Response) => {
+  res.redirect(req.buildUrl(`/publish/${req.params.datasetId}/overview`, req.language));
```

Comment:
Default page from the dataset list is now overview, so makes sense that a broken dataset url redirects there I think.